lsp4clj/src/clojure_lsp/lsp.clj

```diff
@@ -0,0 +1,436 @@
+(ns clojure-lsp.lsp
```

Comment:
IMO we could rename all ns inside lsp4clj to `lsp4clj`, which makes sense and helps a lot understand the boundaries between lsp4clj and the rest of clojure-lsp, WDYT?
```suggestion
(ns lsp4clj.core
```

Reply:
I named it lsp4clj.lsp. I really really dislike the core naming convention. I always have 10 core.clj files open and requiring them is also a pain since you can't just `:as` the top-level namespace. I like the way @seancorfield does it in all his libs, e.g. https://github.com/seancorfield/next-jdbc/tree/develop/src/next.
So have your main entry ns (usually this would have been core) a level higher than the implementation ns's and give it the name of the lib. So I probably should change the ns to lsp4clj.lsp4clj. I'd also be happy switching to the more modern clojars format of something like `org.clojure-lsp.lsp4j` or what ever you'd want to call the org

Reply:
the org is `com.github.io` a long name IMO, I still prefer something that indicate that this is the entrypoint of the lsp4j without repeating the name, I already used Sean suggestion on deps-bin but for this one IMO I still think  lsp4clj.core looks better

Reply:
done